Tests/LoggingTests/LoggingTest.swift

```diff
@@ -889,3 +889,13 @@ extension Logger {
     }
     #endif
 }
+
+// Sendable
+
+#if compiler(>=5.6)
+// @unchecked Sendable since not thread-safe by definition
+extension LoggingTest.LazyMetadataBox: @unchecked Sendable {}
```

Comment:
We should not mark things that are `"not thread-safe by definition"` as Sendable, should we?

Reply:
what do you have in mind to deal with this?

Reply:
If it's not used in a thread-crossing context, why does it need to be `Sendable` at all?

Reply:
Ok in this case I think it might be best to explain that in this comment.